code/game/objects/items/weapons/storage/judgement_sheath.dm

```diff
@@ -0,0 +1,169 @@
+	. = ..()
+	if(. && istype(W, /obj/item/tool/sword/katana/spatial_cutter))
+		var/obj/item/tool/sword/katana/spatial_cutter/cutter = W
+		cutter.ActivateSpatialCuts(src, user)
+/obj/item/storage/belt/sheath/judgement/afterattack(atom/target, mob/user, proximity_flag, params)
+	. = ..()
+	var/turf/T = get_turf(target)
+	if(RiftTarget)
+		if(proximity_flag && world.time >= next_rift)
+			
+			var/spatialCutsCount = 0
+			var/cutcolor = "#ffffff"
+			for(var/atom/movable/SpatialCut/C in T)
+				spatialCutsCount++
+				cutcolor = C.color
+				qdel(C)
+			if(!spatialCutsCount)
+				RiftTarget = null
+			else if(spatialCutsCount == 2)
+				playsound(T, 'sound/effects/portal_open.ogg', 100, extrarange = 3, ignore_walls = FALSE, use_pressure = FALSE)
+				var/obj/effect/portal/P = new/obj/effect/portal/single_use{icon = 'icons/obj/spatial_cut.dmi';}(T)
+				P.color = cutcolor
+				P.set_target(RiftTarget)
+				next_rift = world.time + RiftCooldown
+		else
+			RiftTarget = null
+	else if(!T.density)
+		RiftTarget = get_turf(target)
+		to_chat(user, "\The [src] shines.")
+
+
+/obj/item/storage/belt/sheath/judgement/equiped/New()
+	new/obj/item/tool/sword/katana/spatial_cutter/yamato(src)
+	. = ..()
+
+/* I NEED MORE POWER
+/obj/item/storage/belt/sheath/judgement/attackby(obj/item/I, mob/living/user, params)
+	if(nt_sword_attack(I, user))
+		return
+	. = ..()
+*/
+/obj/item/tool/sword/katana/spatial_cutter
+	desc = "This is a strange katana, when you move it in the air you can see something like shadow of it following it."
+	spawn_blacklisted = TRUE
+	var/LifeTimeOfSpatialCuts = 5 SECONDS
+
+	var/list/SpatialCuts = list()
+	var/SpatialCutsColor = "#ffffff"
+	var/SpatialCutsType = /atom/movable/SpatialCut
+
+	var/tmp/next_spatial_cut = 0
+
+	var/SpatialCutCooldown = 0.5 SECOND
+
+	embed_mult = 0
+
+	afterattack(atom/target, mob/user, proximity_flag, params)
+		. = ..()
+		if(proximity_flag && world.time >= next_spatial_cut)
+			SpatialCuts.Add(new SpatialCutsType(get_turf(target), src, SpatialCutsColor, rand(0, 36) * 10, LifeTimeOfSpatialCuts))
+			next_spatial_cut = world.time + SpatialCutCooldown
+	proc/ActivateSpatialCuts(obj/item/storage/belt/sheath/judgement/sheath, mob/user)
+		. = length(SpatialCuts)
+		for(var/atom/movable/SpatialCut/C in SpatialCuts)
+			spawn(0)
+				C.Activate(user)
+/obj/item/tool/sword/katana/spatial_cutter/yamato
+	name = "yamato"
+	desc = "This is a strange katana, when you move it in the air you can see something like shadow of it following it. When you look at it you almost hear something like 'I need more power!'."
+	icon_state = "yamato"
+	SpatialCutsColor = "#66aaff"
+	ActivateSpatialCuts(obj/item/storage/belt/sheath/judgement/sheath)
+		. = ..()
+		if(.)
+			var/quote = "<b>Voice from somewhere</b>, says '[pick("Too slow.", "You are finished!", "It's over!")]'"
+			audible_message(quote, "You almost can hear someone's voice.", 3)
```

Comment:
How does this entire part appears in game? Wouldn't it appear "triple-quoted", as in `'"text"'`?

Reply:
What?

Reply:
![image](https://user-images.githubusercontent.com/46600554/131292538-51455495-af41-41b2-9440-6512bb469d57.png)